### ISSUE TITLE
Server is hitting LOVD API and Not CLI in some cases

### ISSUE THREAD

**Issue by Peter-J-Freeman** [openvar/variantValidator#727]
**Describe the bug**
See title

**To Reproduce**
@ifokkema to send a few examples

**Expected behavior**
Should not trigger the API unless there is a fail

**Additional context**
Increase test coverage for LOVD HGVS checker usage

**Comment by Peter-J-Freeman** [comment]
One thought I have is whether it is a batch issue. Perhaps the CLI gets overloaded so fires an error???

**Comment by ifokkema** [comment]
Requests do seem to arrive mostly in bursts, but not always.
I had left a few examples around here somewhere on GitHub yesterday; let me see if I can find back the issue where I left it.

**Comment by ifokkema** [comment]
Found it:

Sure; raw requests:
```
143.210.56.69 [30/Apr/2025:14:49:06 +0200] "GET /v2/checkHGVS/chr2:21249716C%3ET HTTP/1.1" 200 4895 "-" "python-requests/2.31.0"
143.210.56.69 [30/Apr/2025:14:49:32 +0200] "GET /v2/checkHGVS/GRCh37:2:21249716C%3ET HTTP/1.1" 200 4792 "-" "python-requests/2.31.0"
143.210.56.69 [30/Apr/2025:14:49:46 +0200] "GET /v2/checkHGVS/ENST00000316448.5:c.1099_1150del HTTP/1.1" 200 4436 "-" "python-requests/2.31.0"
143.210.56.69 [30/Apr/2025:14:49:53 +0200] "GET /v2/checkHGVS/ENST00000316448.5:c.1099_1150del HTTP/1.1" 200 4435 "-" "python-requests/2.31.0"
143.210.56.69 [30/Apr/2025:14:51:16 +0200] "GET /v2/checkHGVS/chr2-201639057%20G%3EGA HTTP/1.1" 200 4295 "-" "python-requests/2.31.0"
143.210.56.69 [30/Apr/2025:14:53:09 +0200] "GET /v2/checkHGVS/chr20:3891442T%3EG HTTP/1.1" 200 4890 "-" "python-requests/2.31.0"
143.210.56.69 [30/Apr/2025:14:54:33 +0200] "GET /v2/checkHGVS/NM_000527.5 HTTP/1.1" 200 3877 "-" "python-requests/2.31.0"
143.210.56.69 [30/Apr/2025:15:01:07 +0200] "GET /v2/checkHGVS/chr13%0923279452%0923279452%09T%09C HTTP/1.1" 200 4319 "-" "python-requests/2.31.0"
```

The list of "variants" extracted from that:

```
chr2:21249716C>T
GRCh37:2:21249716C>T
ENST00000316448.5:c.1099_1150del
ENST00000316448.5:c.1099_1150del
chr2-201639057 G>GA
chr20:3891442T>G
NM_000527.5
chr13	23279452	23279452	T	C
```

I see no "normal" variants there (except for the Ensembl reference sequence ones), which may suggest we indeed don't get everything, but scrolling through the logs, I do see some valid-looking entries as well.

**Comment by ifokkema** [comment]
And from earlier today; raw:
```
143.210.56.69 - - [01/May/2025:11:49:23 +0200] "GET /v2/checkHGVS/NM_000492.4:c.7A%3ET HTTP/1.1" 200 4399 "-" "python-requests/2.31.0"
143.210.56.69 - - [01/May/2025:11:49:23 +0200] "GET /v2/checkHGVS/NM_000492.4:c.9del HTTP/1.1" 200 4402 "-" "python-requests/2.31.0"
143.210.56.69 - - [01/May/2025:11:49:23 +0200] "GET /v2/checkHGVS/NM_000492.4:c.8G%3ET HTTP/1.1" 200 4401 "-" "python-requests/2.31.0"
143.210.56.69 - - [01/May/2025:11:49:23 +0200] "GET /v2/checkHGVS/NM_000492.4:c.11C%3ET HTTP/1.1" 200 4404 "-" "python-requests/2.31.0"
143.210.56.69 - - [01/May/2025:11:49:23 +0200] "GET /v2/checkHGVS/NM_000492.4:c.11C%3EA HTTP/1.1" 200 4403 "-" "python-requests/2.31.0"
143.210.56.69 - - [01/May/2025:11:49:23 +0200] "GET /v2/checkHGVS/NM_000492.4:c.12G%3EC HTTP/1.1" 200 4403 "-" "python-requests/2.31.0"
143.210.56.69 - - [01/May/2025:11:49:23 +0200] "GET /v2/checkHGVS/NM_000492.4:c.12G%3ET HTTP/1.1" 200 4405 "-" "python-requests/2.31.0"
143.210.56.69 - - [01/May/2025:11:49:23 +0200] "GET /v2/checkHGVS/NM_000492.4:c.13C%3ET HTTP/1.1" 200 4404 "-" "python-requests/2.31.0"
143.210.56.69 - - [01/May/2025:11:49:23 +0200] "GET /v2/checkHGVS/NM_000492.4:c.14C%3EG HTTP/1.1" 200 4405 "-" "python-requests/2.31.0"
143.210.56.69 - - [01/May/2025:11:49:23 +0200] "GET /v2/checkHGVS/NM_000492.4:c.14C%3ET HTTP/1.1" 200 4403 "-" "python-requests/2.31.0"
143.210.56.69 - - [01/May/2025:11:49:23 +0200] "GET /v2/checkHGVS/NM_000492.4:c.15del HTTP/1.1" 200 4407 "-" "python-requests/2.31.0"
143.210.56.69 - - [01/May/2025:11:49:23 +0200] "GET /v2/checkHGVS/NM_000492.4:c.15T%3EC HTTP/1.1" 200 4404 "-" "python-requests/2.31.0"
143.210.56.69 - - [01/May/2025:11:49:23 +0200] "GET /v2/checkHGVS/NM_000492.4:c.16C%3ET HTTP/1.1" 200 4404 "-" "python-requests/2.31.0"
143.210.56.69 - - [01/May/2025:11:49:23 +0200] "GET /v2/checkHGVS/NM_000492.4:c.16C%3EG HTTP/1.1" 200 4403 "-" "python-requests/2.31.0"
143.210.56.69 - - [01/May/2025:11:49:23 +0200] "GET /v2/checkHGVS/NM_000492.4:c.19G%3ET HTTP/1.1" 200 4403 "-" "python-requests/2.31.0"
143.210.56.69 - - [01/May/2025:11:49:23 +0200] "GET /v2/checkHGVS/NM_000492.4:c.23A%3EG HTTP/1.1" 200 4405 "-" "python-requests/2.31.0"
```

List:

```
NM_000492.4:c.7A>T
NM_000492.4:c.9del
NM_000492.4:c.8G>T
NM_000492.4:c.11C>T
NM_000492.4:c.11C>A
NM_000492.4:c.12G>C
NM_000492.4:c.12G>T
NM_000492.4:c.13C>T
NM_000492.4:c.14C>G
NM_000492.4:c.14C>T
NM_000492.4:c.15del
NM_000492.4:c.15T>C
NM_000492.4:c.16C>T
NM_000492.4:c.16C>G
NM_000492.4:c.19G>T
NM_000492.4:c.23A>G
```

They're likely from a batch check, indeed, but could also be API activity.

**Comment by Peter-J-Freeman** [comment]
Thanks @ifokkema I will test these on my system.

**Comment by Peter-J-Freeman** [comment]
I have added code which re-tries the CLI if there is an unexpected fail. from the second list I cannot see anything that should trip up VV and require LOVD intervention. This is pretty odd for list 2

Will test the 1st list

**Comment by Peter-J-Freeman** [comment]
All variants in the 2nd list pass. Nothing sent to the LOVD CLI or other. Weird!

**Comment by ifokkema** [comment]
Hmmm.... New theory: I see two unique IP addresses sending VV input. One sends a lot of requests, the other very few. Is there another system running that still uses the API a lot, while the new code uses the API very little?

**Comment by Peter-J-Freeman** [comment]
I have 3 systems using LOVD. My machine, 2 servers. can you email me the IP addresses? Thanks

**Comment by Peter-J-Freeman** [comment]
The lists on this page are the interactive server vval.org!

**Comment by Peter-J-Freeman** [comment]
I know that this is where the batch tool is running. But I also know that the CLI is installed. If its bursts, then it is likely the batch tool.

**Comment by Peter-J-Freeman** [comment]
Have now narrowed this to apache setup. Apache is not finding a PhP package

**Comment by ifokkema** [comment]
Perhaps using an absolute or relative path in the Python code to the executable may help? Otherwise, setting `PATH` before execution somewhere should also help.

**Comment by Peter-J-Freeman** [comment]
I have left it with IT services. Fingers crossed. They usually solve things
